IBPSA/BoundaryConditions/BESTEST.mo

```diff
@@ -0,0 +1,1718 @@
+          points={{-58,12},{0,12},{0,78},{-58,78}},
+          color={255,204,51},
+          thickness=0.5));
+      connect(Azi000Til30.weaBus, Azi315Til90.weaBus) annotation (Line(
+          points={{-58,-18},{0,-18},{0,78},{-58,78}},
+          color={255,204,51},
+          thickness=0.5));
+      connect(Azi090Til30.weaBus, Azi315Til90.weaBus) annotation (Line(
+          points={{-58,-46},{0,-46},{0,78},{-58,78}},
+          color={255,204,51},
+          thickness=0.5));
+      connect(weaBus.pAtm, x_pTphi.p_in) annotation (Line(
+          points={{-1,-68},{-34,-68},{-34,-74},{-46,-74}},
+          color={255,204,51},
+          thickness=0.5), Text(
+          string="%first",
+          index=-1,
+          extent={{-3,-6},{-3,-6}},
+          horizontalAlignment=TextAlignment.Right));
+      connect(weaBus.TDryBul, x_pTphi.T) annotation (Line(
+          points={{-1,-68},{-22,-68},{-22,-80},{-46,-80}},
+          color={255,204,51},
+          thickness=0.5), Text(
+          string="%first",
+          index=-1,
+          extent={{6,3},{6,3}},
+          horizontalAlignment=TextAlignment.Left));
+      connect(weaBus.relHum, x_pTphi.phi) annotation (Line(
+          points={{-1,-68},{-18,-68},{-18,-86},{-46,-86}},
+          color={255,204,51},
+          thickness=0.5), Text(
+          string="%first",
+          index=-1,
+          extent={{6,3},{6,3}},
+          horizontalAlignment=TextAlignment.Left));
+      connect(x_pTphi.X[1], toDryAir.XiTotalAir) annotation (Line(points={{-69,
+              -80},{-72,-80},{-72,-81},{-74.9,-81}}, color={0,0,127}));
+      connect(weaDat1.weaBus, weaBusDew) annotation (Line(
+          points={{50,-80},{50,-74},{51,-74},{51,-68},{51,-68}},
+          color={255,204,51},
+          thickness=0.5), Text(
+          string="%second",
+          index=1,
+          extent={{-3,6},{-3,6}},
+          horizontalAlignment=TextAlignment.Right));
+      annotation (
+      Documentation(info="<html>
+<h4>WD100: Base Case</h4>
+<p>Weather data file : 725650.epw</p>
+<p><i>Table 1: Site Data for Weather file 725650.epw</i></p>
+<table cellspacing=\"2\" cellpadding=\"0\" border=\"1\"><tr>
+<td><p>Latitude</p></td>
+<td><p>39.833&deg; north</p></td>
+</tr>
+<tr>
+<td><p>Longitude</p></td>
+<td><p>104.65&deg; west</p></td>
+</tr>
+<tr>
+<td><p>Altitude</p></td>
+<td><p>1650 m</p></td>
+</tr>
+<tr>
+<td><p>Time Zone</p></td>
+<td><p>7</p></td>
+</tr>
+</table>
+<p><br><br><h4>Additional parameters and correlations:</h4></p>
+<p>Ground reflectance &rho; is set to 0</p>
+<p>Black body temperature calculated using Horizontal radiation or dew point temperature and sky cover</p><p>Perez or Isoentropic sky model used</p>
+</html>",     revisions="<html>
+<ul>
+<li>
+March 11, 2020, by Ettore Zanetti:<br/>
+First implementation.
+</li>
+</ul>
+</html>"),
+    experiment(
+          StopTime=31539600,
+          __Dymola_NumberOfIntervals=35043,
```

Comment:
I was using it as a reference to check wether the extrapolation done by dassl with the adaptive time step and the fixed runge kutta that I am using in the Python scripts gave the same results. It is not needed though, since in the end all the instructions for the simulation are passed in the Python script. I removed it.